<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QgsAddJoinDialogBase</class>
 <widget class="QDialog" name="QgsAddJoinDialogBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>281</width>
    <height>160</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add vector join</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="mJoinLayerLabel">
     <property name="text">
      <string>Join layer</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="mJoinLayerComboBox"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="mJoinFieldLabel">
     <property name="text">
      <string>Join field</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="mJoinFieldComboBox"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="mTargetFieldLabel">
     <property name="text">
      <string>Target field</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QComboBox" name="mTargetFieldComboBox"/>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QCheckBox" name="mCreateIndexCheckBox">
     <property name="text">
      <string>Create attribute index on join field</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QCheckBox" name="mCacheInMemoryCheckBox">
     <property name="text">
      <string>Cache join layer in virtual memory</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>QgsAddJoinDialogBase</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>QgsAddJoinDialogBase</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
